cmd2/argparse_custom.py

```diff
@@ -743,38 +744,46 @@ def _metavar_formatter(self, action, default_metavar) -> Callable:
             # End cmd2 customization
         else:
             result = default_metavar
+        return result
+
+    def _metavar_formatter(self, action, default_metavar) -> Callable:
+        metavar = self._determine_metavar(action, default_metavar)
 
         # noinspection PyMissingOrEmptyDocstring
         def format(tuple_size):
-            if isinstance(result, tuple):
-                return result
+            if isinstance(metavar, tuple):
+                return metavar
             else:
-                return (result, ) * tuple_size
+                return (metavar, ) * tuple_size
         return format
 
     # noinspection PyProtectedMember
     def _format_args(self, action, default_metavar) -> str:
-        get_metavar = self._metavar_formatter(action, default_metavar)
-        # Begin cmd2 customization (less verbose)
-        nargs_range = getattr(action, ATTR_NARGS_RANGE, None)
+        """Customized to handle ranged nargs and make other output less verbose"""
+        metavar = self._determine_metavar(action, default_metavar)
```

Comment:
We're calling `self._determine_metavar()` twice in a row here. Maybe pass it into `self._metavar_formatter()` or have `self._metavar_formatter()` return a tuple?

Reply:
`self._metavar_formatter()` is an overrided `argparse` function. I can't change its signature.